### PR TITLE
Update class for protobuf on new triggering/execution system

### PR DESCRIPTION
**Pull request by v9n**
Sync the typescript with our protobuf

### CODE REVIEW

**Comment by v9n** [comment]
@chrisli30 Please review and ignore the staging error

File: grpc_codegen/avs.proto [hunk 1]
```diff
@@ -32,13 +32,42 @@ message CheckinResp {
...
 
+enum MessageOp {
+  Unset              = 0;
+  MonitorTaskTrigger = 1;
+  CancelTask         = 2;
+  DeleteTask         = 3;
+  CompletedTask      = 4;
+}
+
+message SyncMessagesResp {
```
Comment (v9n) [inline grpc_codegen/avs.proto:50]: please skip these changes. we're split the protobuf into node vs user feature in next phase.

File: src/task.ts [hunk 2]
```diff
@@ -33,8 +33,11 @@ class Task implements TaskType {
     this.executions = task.getExecutionsList().map(execution => {
        return {
          epoch: execution.getEpoch(),
-         userOpHash: execution.getUserOpHash(),
```
Comment (v9n) [inline src/task.ts:36]: we no longer has userOpHash. 

We just call it outputData as a generic name for outcome (result) of a node execution.

File: grpc_codegen/avs.proto (hunk 1 above)
Comment (chrisli30) [reply grpc_codegen/avs.proto:50]: Got it yes, I see this `SyncMessagesResp` widely used in the avs_pb.js now. 👌

File: src/task.ts (hunk 2 above)
Comment (chrisli30) [reply src/task.ts:36]: 👌

**Review by chrisli30 — APPROVED** [review]
Looks good and all changes have been picked up in ava-sdk

**Review by chrisli30 — APPROVED** [review]
Looks good and changes have been rebased and picked up in #41

**Pull request merged**